lib/src/task.dart

```diff
@@ -0,0 +1,59 @@
+class Task {
+  Task({
+    this.status,
+    this.updateId,
```

Comment:
```suggestion
    this.uid,
```
This field should be renamed as in the `json` structure to avoid confusion, no?

Reply:
Sorry, but what do you mean about the `json` structure? 🤔

Reply:
But maybe it wasn't a good term sorry

Reply:
Oh, now I get it, yes I've updated that term already thanks ;)